Makefile

```diff
@@ -69,9 +69,9 @@ tools-linux-amd64:
 $(foreach SERVICE,$(SERVICES),$(eval $(SERVICE_RULES)))
 $(foreach TOOL,$(TOOLS),$(eval $(TOOL_RULES)))
 
-install-vendor: .gitmodules
-	@echo Updating submodules
-	git submodule update --init --recursive
+install-vendor: install-glide
```

Comment:
Yeah think it's as simple as setting `license_dir` on line 14 to `.ci/uber-licence`